app/views/layouts/_header.html.erb

```diff
@@ -4,6 +4,13 @@
     <ul>
       <li><%= link_to 'Upcoming Events', upcoming_events_path %></li>
       <li><%= link_to 'New Presentation', new_presentation_path %></li>
+      <li>
+        <% if current_person %>
+          <%= link_to 'Sign Out', signout_path %>
+        <% else %>
+          <%= link_to 'Sign In', '/auth/google_oauth2' %>
```

Comment:
I think the gem adds that route in for us or something. It's not defined in our routes but I've tested it and it works / this is old behaviour from the google oauth PR being re introduced.